src/user/schema/user.schema.ts

```diff
@@ -0,0 +1,44 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export class Location {
+  @Prop()
+  provided: string;
+
+  @Prop()
+  lat: number;
+
+  @Prop()
+  long: number;
+}
+
+@Schema({ _id: false, timestamps: false })
+export class GithubProfile extends Document {
+  @Prop({ required: true })
+  _id: string;
+
+  @Prop({ type: Location })
+  location: Location;
```

Comment:
In my opinion this needs to go to the client. There might be the scenario where someone updates this and then its not accurate here. We should store this redundantly. When making the API-Call the client needs to call the github-api to get the location.